client/src/pages/MyRoadmaps/Card.jsx

```diff
@@ -5,7 +5,7 @@ const Card = ({ bannerImage, id, title, tagline, progress = 0 }) => {
   return (
     <Link to={`/myroadmaps/${id}`}>
       <section className="roadmap-card">
-        <img src={bannerImage} alt="" />
+        <img src={bannerImage} alt="My Roadmap banner" />
```

Comment:
please use the `title` as the alt tag here. The title is available through the props

Reply:
Thank you for the review and suggestion @siddhigate! I have implemented the requested changes.